### PR TITLE
Prevent overlapping of Images On Home Screen

### LINKED ISSUE #142: Use Layout_weight to prevent overlapping of Images On Home Screen
**Summary:** 

In Preview of activity_main, we could use layout_weight to arrange the icons symmetrically according the Phone's Screen. Right Now, the images may overlap if the screen size is small.

Add logcat files here (if possible).

Instead of this:

android:layout_width="@dimen/margin_178dp"
        android:layout_height="@dimen/margin_205dp"

we could use this

 android:layout_width="@dimen/margin_178dp"
            android:layout_weight="1"
            android:layout_height="@dimen/margin_205dp"

**Device and Android version:** 

Model: Xiaomi MiA1
Android: 9.0 Pie
ROM: Stock ROM
 
**Screen-shots:** 

This is what it look like in activity_main preview(Look at the icons gaps in both photos)

Screenshot from 2019-03-26 01-39-43

Now check this Image

WhatsApp Image 2019-03-26 at 1 42 45 AM

**Would you like to work on the issue?**

Yes, I would like to work on this issue.

### PR DESCRIPTION
**Pull request by guy34code**
**Description (required)**

Fixes #142 {Use Layout_weight to prevent overlapping of Images On Home Screen} 

**Screenshots showing what changed (optional - for UI changes)**

**Before the Changes:**

WhatsApp Image 2019-03-26 at 7 57 41 PM

**After Changes:**

WhatsApp Image 2019-03-26 at 10 38 28 PM

---

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -31,11 +31,13 @@
         android:id="@+id/files"
         android:layout_width="@dimen/margin_178dp"
         android:layout_height="@dimen/margin_205dp"
+        android:layout_marginStart="8dp"
```
Comment (dev-ritik) [inline app/src/main/res/layout/activity_main.xml]: Use `@dimen/margin_8dp` instead

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -56,12 +58,13 @@
         android:id="@+id/rcvdImages"
         android:layout_width="@dimen/margin_178dp"
         android:layout_height="@dimen/margin_205dp"
-        android:layout_marginBottom="@dimen/margin_8dp"
+        android:layout_marginStart="8dp"
```
Comment (dev-ritik) [inline app/src/main/res/layout/activity_main.xml]: Use `@dimen/margin_8dp` instead

Comment (guy34code) [reply app/src/main/res/layout/activity_main.xml]: Okay, please review the new activity_main

**Comment by guy34code** [comment]
@dev-ritik  I accidently commit the changes to fix #143 under the same pull request.
I'm new to open source and don't know much, Is there any way to create new pull request now for issue #143 ?

**Comment by dev-ritik** [comment]
@guy34code I would suggest you to create a new branch for #143 
On the new branch, drop the commits of #142 while u rebase it.
Make a new pull request for it.
On the existing branch, make a hard reset to the extra commits.
Force push the existing branch to update the PR.

**Pull request closed without merging**